OracleWebCenterSites/README.md

```diff
@@ -0,0 +1,430 @@
+```   
+   **Note**: Container name can be given only if the container is located on the same host machine. Ensure SERVICE_NAME is a valid PDB service name in your database as given in the $ORACLE_HOME/admin/ORCLCDB/tnsnames.ora file.
+   
+For Additional information on [running Oracle Database image](https://container-registry.oracle.com). Click **Home > Database > enterprise**.
+
+For monitoring Docker container Logs:
+```
+    $ docker logs -f --tail 900 WCSites12212Database
+```
+
+## 7. Running Oracle WebCenter Sites Docker Container
+To run the Oracle WebCenter Sites Docker container, you need to create:
+* Admin container to manage the admin server. The admin container performs RCU, ConfigWizard, and WebCenter Sites Configuration processes. 
+* Managed container to manage the managed server.
+
+### A. Creating Admin container for WebCenter Sites Admin Server
+This container is used to manage Admin Server.
+
+#### 1. Update the environment file
+`wcsitesadminserver.env.list` is located at `../docker-images-master/OracleWebCenterSites/dockerfiles/` that contains parameters that are passed to WebLogic admin server. Update this file with the information pertinent to your environment:
+
+`wcsitesadminserver.env.list` file details:
+```
+    DOCKER_HOST=<Hostname>
+    DB_CONNECTSTRING=<Hostname/ContainerName>:<Database Port>/<DB_PDB>.<DB_DOMAIN>
+    DB_USER=<By default: sys>
+    DB_PASSWORD=<Database Password>
+    RCU_PREFIX=<RCU_PREFIX>
+    DB_SCHEMA_PASSWORD=<database_schema_password: if not provided, it gets auto generated>
+    SAMPLES=<To install sample Sites, set samples as true, else set as false>
+    DOMAIN_NAME=<Domain Name optional>
+    SITES_SERVER_NAME=<Sites Server Name optional>
+    ADMIN_USERNAME=<Admin UserName, default: weblogic>
+    ADMIN_PASSWORD=<Admin_Password: if not provided, it gets auto generated>
+```
+#### 2. Start the Admin Container
+
+a. To run WebLogic Admin server container, go to `dockerfiles` folder located at `../docker-images-master/OracleWebCenterSites/`. 
+
+b. Run the following command and pass the environment file name as a parameter: 
+
+```
+   $ docker run -d --name <container_name> --network=<network_name> -p <weblogic_port>:7001 -p <weblogic_ssl_port>:9001 -v <user_projects_volume_dir>:/u01/oracle/user_projects -v <sites_shared_volume_dir>:/u01/oracle/sites-shared --env-file <environment_file> <repo_name:tag_name>
+```
+Sample command:
+```
+   $ docker run -d -t --name WCSitesAdminContainer --network=WCSitesNet -p 7001:7001 -p 9001:9001 -v /scratch/DockerVolume/WCSitesVolume/WCSites:/u01/oracle/user_projects -v /scratch/DockerVolume/WCSitesVolume/WCSitesShared:/u01/oracle/sites-shared --env-file ./wcsitesadminserver.env.list oracle/wcsites:12.2.1.3
+```
+Admin Container start up command explained:
+```
+   | --name                 | container_name          | Database name; set to ‘WCSitesAdminContainer’                                             |
```

Comment:
If you remove the indent from these lines, Markdown would create a proper table. You can then use back ticks inside the table to format it monospaced.